packages/spacecat-shared-google-client/src/index.js

```diff
@@ -108,6 +111,8 @@ export default class GoogleClient {
 
     await this.#refreshTokenIfExpired();
 
+    const auditUrl = await composeAuditURL(this.baseUrl);
```

Comment:
fyi: this will probably not work for customers with bot detection: https://cq-dev.slack.com/archives/C05A45JBP9N/p1715100249734359?thread_ts=1715068794.711109&cid=C05A45JBP9N